### PR TITLE
Implemented Login, Register, Authenticate services (#21)

### PR DESCRIPTION
**Pull request by yasir18**

### CODE REVIEW

**Comment by asif-ir** [comment]
There are some meaningless tests which area failing.  Can you fix the tests along with this?

File: src/config/jwtConfig.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  JWT_SECRET: "jwt_secret",
```
Comment (asif-ir) [inline src/config/jwtConfig.js]: Change to JWT_SECRET: process.env.JWT_SECRET || "jwt_secret",

File: src/services/auth-service.js [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+  }
+
+  async login(username, password) {
+    const result = await User.get(username);
+    if (result && result.length > 0) {
+      const hashedPassword = result[0].password;
+      const isCorrectPwd = bcrypt.compareSync(password, hashedPassword);
+      if (isCorrectPwd) {
+        return true;
+      } else return false;
```
Comment (asif-ir) [inline src/services/auth-service.js]: This else block not needed I guess.  Simply `return isCorrectPwd; `

Comment (yasir18) [reply src/services/auth-service.js]: Done

File: src/config/jwtConfig.js (hunk 1 above)
Comment (yasir18) [reply src/config/jwtConfig.js]: Done

**Pull request merged**